src/Servers/IIS/IISIntegration/src/IISHostingStartup.cs

```diff
@@ -7,8 +7,18 @@
 
 namespace Microsoft.AspNetCore.Server.IISIntegration
 {
+    /// <summary>
+    /// The <see cref="IHostingStartup"/> to add IISIntegration to apps.
+    /// </summary>
     public class IISHostingStartup : IHostingStartup
     {
+        /// <summary>
+        /// Configures the middleware pipeline.
```

Comment:
Can we expand on what this means? Like it configures your web app to use out of process IIS integration?